manim/scene/scene_file_writer.py

```diff
@@ -463,9 +463,6 @@ def finish(self):
         frame in the default image directory.
         """
         if write_to_movie():
-            if hasattr(self, "writing_process"):
-                self.writing_process.terminate()
-
```

Comment:
@JasonGrace2282 

I think this is a remnant from the time when manim opened a subprocess to write frames to `FFmpeg` via stdin. It didn't hurt anything, but I figure it's no longer necessary. I took it out in one of the commits just now.